### PR TITLE
Files related to upload csv feature

### PR DESCRIPTION
**Pull request by zfred219**
- User can create new users, orgs by uploading csv
- Some CSV uploading UI improvement

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -18,7 +19,6 @@ gem 'jbuilder', '~> 2.0'
 gem 'bootstrap_form'
 gem 'paperclip', '~> 5.2.1'
 gem 'aws-sdk', '~> 2.3.0'
-
 group :development do
   # Access an IRB console on exception pages or by using <%= console %> in views
   gem 'web-console', '~> 2.0'
```
Comment (EnjiaXu) [inline Gemfile:24]: Ignore this file (Included in the first PR).

File: Gemfile.lock [hunk 2]
```diff
@@ -372,6 +374,7 @@ DEPENDENCIES
   jquery-rails
   json
   launchy
+  mimemagic (~> 0.3.10)
   omniauth-github
   paperclip (~> 5.2.1)
   pg (< 1.0.0)
```
Comment (EnjiaXu) [inline Gemfile.lock:380]: Ignore this file (Included in the first PR).

File: features/public_search.feature [hunk 3]
```diff
@@ -24,13 +24,13 @@ Background: Logged Out
...
-Scenario: search for an app by name keyword
-    Given I publicly search for "1"
-    Then I should see "app 1"
-    But I should not see "app 2"
-    And I should not see "app 3"
+#Scenario: search for an app by name keyword
+   # Given I publicly search for "1"
+    #Then I should see "app 1"
+   # But I should not see "app 2"
+    #And I should not see "app 3"
```
Comment (EnjiaXu) [inline features/public_search.feature:36]: Ignore this file (Included in the first PR).

File: app/controllers/orgs_controller.rb [hunk 4]
```diff
@@ -87,6 +87,15 @@ def destroy
...
 
+  def import
+    begin
+      Org.import(params[:file])
+      redirect_to orgs_path, notice: 'Org was successfully imported.'
+    rescue StandardError => e  
+      redirect_to orgs_path, notice: e.message
+    end
+  end
+
```
Comment (zfred219) [inline app/controllers/orgs_controller.rb:98]: Only part related to this feature

File: config/routes.rb [hunk 5]
```diff
@@ -14,12 +31,20 @@
...
     resources :comments, :only => [:create, :update], module: :orgs
+    collection {post :import}
   end
   resources :comments, :only => [:edit, :destroy]
+
+  get 'users/import' => 'users#import'
+
   resources :users do
     resources :comments, only: [:create, :update], module: :users
+    collection {post :import}
```
Comment (zfred219) [inline config/routes.rb:47]: Uploading CSV related routes

File: app/assets/stylesheets/application.scss [hunk 6]
```diff
@@ -7,4 +7,10 @@
...
 @import "comments";
-@import "iterations";
\ No newline at end of file
+@import "iterations";
+@import "users";
+
+body{
+    margin-top: 0px;
+    padding: 0px;
+}
```
Comment (EnjiaXu) [inline app/assets/stylesheets/application.scss:16]: Ignore this file (already in a previous PR)

File: config/initializers/assets.rb [hunk 7]
```diff
@@ -9,3 +9,8 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
+# Rails.application.config.assets.precompile += %w( main_page.js )
+Rails.application.config.assets.precompile += %w( apps.scss )
+Rails.application.config.assets.precompile += %w( matching.css )
+Rails.application.config.assets.precompile += %w( progress.css )
+Rails.application.config.assets.precompile += %w( navbar.css )
```
Comment (EnjiaXu) [inline config/initializers/assets.rb:16]: Ignore this file (already in a previous PR)

**Pull request merged**